### DISCUSSION TITLE
HP : Quel est l'impact environnemental de votre site ?

### DISCUSSION THREAD

**Discussion by margaux-escande** [cnumr/EcoIndex#149] [Bug]
Pour être plus exact comme il y a plusieurs critères environnementaux : 
- quels sont les impacts environnementaux de votre site 
- ou 
- quelle est l'empreinte environnementale de votre site

**Comment by DocRoms** [comment]
Je suis plutôt en accord avec cette pertinente remarque ! @aureliebaton tu en penses quoi ?

**Comment by aureliebaton** [comment]
Oui c'est une bonne remarque ! 

Je vote pour "Quelle est l'empreinte environnementale de votre site ?" 

---

Et par contre pour le bouton "Tester le site" on pourrait mettre "Tester la page" ce qui est plus correct car on fait l'analyse sur la page et pas sur le site entier. Qu'en pensez-vous @margaux-escande @DocRoms @yaaax

**Reply by DocRoms** [reply]
Oui, c'est également une bonne idée ! On reste précis !

**Reply by yaaax** [reply]
On crée une issue ?
